### ISSUE TITLE
hero-unit component not currently able to use retina and non retina images

### ISSUE THREAD

**Issue by ScottDowne** [mozilla/teach.webmaker.org#307]
```
  render: function() {
    return 
        <HeroUnit image="/img/hero-unit.jpg">
          <h1>Teach like Mozilla</h1>
          <h2>We learn best by making & reflecting, together.</h2>
        </HeroUnit>;
  }
```

Something like this, if I have two images, and hero unit could support both images, it would be awesome.

I would like to use it, because it does a bunch that I need.

Internally it used background-image and not sure how to make that work with retina, but if I can modify hero unit to do this I'm going to submit a pr.

**Comment by toolness** [comment]
Was this fixed by #309?

**Comment by ScottDowne** [comment]
Yup, fixed by #309!

It uses a bit of the like mozilla page as an example of using hero unit with retina, but the like mozilla page isn't complete yet.
